soccer/planning/planner/InterceptPlanner.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "Planner.hpp"
```

Comment:
This is almost a direct port from the original intercept planner. There should be no major functional changes.